public/scripts/components/sub-components/StudentDropdown.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import students from './../../collections/StudentsCollection.js'; 
+
+export default React.createClass({
+	getInitialState: function() {
+		return {
+			students: students
+		};
+	},
+	componentDidMount: function() {
+		// this.state.students.fetch();
+		console.log(this.state)
+		students.on('update', () => {
+			this.setState({students: students});
+		});
+		students.fetch();
+		console.log(this.state.students);
+	},
+	render: function() {
+		const allStudents = this.state.students.map((student, index, array) => {
+			console.log(student)
+		return (
+			<option value="Student" key={index}>{student.get('firstName')} {student.get('lastName')}</option> 
+			)
+		});
+		return (
+		<div className="student-dropdown-component">
+			<p className="student-prompt">Which student is participating?</p>
+			<div align="center">
+				<select name="mydropdown">
+					<option value="Pick">Pick a student</option>
+					{allStudents}
+				</select>
+			</div>
+		</div>
+		);
+	}
+});
+
```

Comment:
In order to be able to get the selected student at some point in the future you'll need to do one of two things:

1. Move this logic out of its own component into the Details component
2. Add an onChange callback prop that gets called each time the selection is changed.

I think the first one would be easier. Let me know if you have questions about this.